Dockerfile

```diff
@@ -11,15 +11,13 @@ COPY requirements_test.txt .
 COPY ocrd_tesserocr ./ocrd_tesserocr
 COPY Makefile .
 RUN apt-get update && \
-    apt-get install -y software-properties-common && \
-    add-apt-repository ppa:alex-p/tesseract-ocr && \
-    apt-get update && \
-    apt-get -y install --no-install-recommends \
-    libtesseract-dev \
+    apt-get install -y --no-install-recommends software-properties-common && \
```

Comment:
Would it make sense to have `software-properties-common` in `ocrd/core` Dockerfile?

Reply:
c.f. OCR-D/core#368

Reply:
It's required here only to provide `add-apt-repository` which is rarely used otherwise.

Reply:
True, no other project currently uses apt-add-repository. But adding `software-properties-common` pulls in glib libraries, so we could omit the `libglib-2.0.0` dep. Space-wise I doubt that the size difference makes much difference.